.changeset/blue-moons-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"apps-rendering-api-models": minor
```

Comment:
I have a feeling this might be a `major` version bump, because we're adding a `required` field. I find this area of semver a bit confusing, but my reasoning is that although its part of an "optional" struct, any existing users of this struct would find this to be a breaking change. Very possible I'm thinking about this in the wrong way though, so I'm interested to hear your thoughts!